### ISSUE TITLE
Minecarts become unlinked going through portals and carts left behind on 1.21

### ISSUE THREAD

**Issue by jawaulk** [constellation-mc/Linkart#49]
### What happened?

When going through a nether portal only the first cart i am riding in goes through and any that are linked are left behind in the overworld. 

### Version

5.4.5 (1.20.6) on MC 1.21

**Comment by jawaulk** [comment]
Since riding minecarts through nether portals was just added in 1.21, I was not sure if I am actually supposed to be able to ride them through the portal linked together with this mod or not. If this should of been a feature request instead of a bug report my apologies. Still loving the mod without the portal support.

Forgot to mention this is on a vanilla client but the server does have a couple other mods on it.

**Comment by zenfyrdev** [comment]
Yes, this is a bug, and I would love to support this, but I can't think of an easy way to do so. There are some major roadblocks, including how the game handles teleportation for non-players and the fact that 1 block in the nether is 8 blocks in the overworld.
